pinot-common/src/main/java/org/apache/pinot/common/metrics/MinionMeter.java

```diff
@@ -31,7 +31,13 @@ public enum MinionMeter implements AbstractMetrics.Meter {
   NUMBER_TASKS_FAILED("tasks", false),
   NUMBER_TASKS_FATAL_FAILED("tasks", false),
   SEGMENT_UPLOAD_FAIL_COUNT("segments", false),
-  SEGMENT_DOWNLOAD_FAIL_COUNT("segments", false);
+  SEGMENT_DOWNLOAD_FAIL_COUNT("segments", false),
+  SEGMENTS_DOWNLOADED("segments", false),
+  SEGMENTS_UPLOADED("segments", false),
+  SEGMENT_SIZE_DOWNLOADED("bytes", false),
+  SEGMENT_SIZE_UPLOADED("bytes", false),
+  RECORDS_PER_SEGMENT("rows", false),
```

Comment:
What does `per` segment stand for? I don't see we are counting the average

Reply:
+1